filebeat/crawler/prospector.go

```diff
@@ -185,6 +185,10 @@ func (p *Prospector) setupHarvesterConfig() error {
 		return err
 	}
 
+	if config.MaxBytes == 0 {
+		config.MaxBytes = cfg.DefaultMaxBytes
+	}
```

Comment:
Should we make it here config.MaxBytes <= 0. We should also log what value was set (will add that). What would happen if we use uint as the variable and some uses -14 in the config?